addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -975,11 +999,14 @@ MockServer.include({
             if (channel.channel_type === 'channel') {
                 delete res.members;
             } else {
-                res['seen_partners_info'] = [{
-                    partner_id: this.currentPartnerId,
-                    seen_message_id: channel.seen_message_id,
-                    fetched_message_id: channel.fetched_message_id,
-                }];
+                res['seen_partners_info'] = res.channel_partner_ids.map(memberId => {
+                    const [cp] = this._getRecords('mail.channel.partner', [['channel_id', 'in', [channel.id]], ['partner_id', '=', memberId]]);
+                    return {
+                        partner_id: memberId,
+                        seen_message_id: cp.seen_message_id,
+                        fetched_message_id: cp.fetched_message_id,
+                    };
+                });
```

Comment:
If the ONLY source of truth for this value is now the channel partner and not the channel, I think all occurrences in tests/mock server of fake field should be removed in this PR. It's not keeping the compatibility of the old syntax now does nothing.

Reply:
Yeah, I did not want to necessarily remove it, but I think it's best just to drop old fake fields altogether.